installer/bundle/bundle_skel.sh

```diff
@@ -1219,7 +1200,6 @@ fi
 
 if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
     compare_install_type
-    compare_arch
```

Comment:
Why remove this check?

Reply:
we no longer distribute x86 packages.

Reply:
This part was not related to the PR, just some clean up of dead code that is called in every install,